.github/CONTRIBUTING.md

```diff
@@ -60,8 +64,8 @@ Once you've found an issue you're interested in tackling, take a look at our [fi
 
 Now that you've selected an issue to work on, you'll need to set up an environment for writing code. We'll assume you already have pip, virtualenv, and git installed and are using a terminal. If not, please set those up before continuing.
 
-1. Clone our repository by executing `git clone git@github.com:ucfopen/canvasapi.git`
-2. Checkout (`git checkout develop`) and then pull the latest commit from the develop branch: `git pull origin develop`
+1. Fork CanvasAPI on GitHub ([see the docs here](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo))
+2. Checkout (`git checkout develop`) and then pull the latest commit from the develop branch: `git pull upstream develop`
 3. Create a new branch with the format **issue/[issue_number]-[issue-title]**: `git checkout -b issue/1-test-issue-for-documentation`
 4. Set up a new virtual environment ( `virtualenv env` ) and activate it (`source env/bin/activate`)
```

Comment:
Should we continue to provide directions for virtualenv or change to pyenv-virtualenv? Regardless, I think linking to external documentation for those tools would be useful

Reply:
Where'd we end up on this after our meeting this morning? Should we link out to virtualenv's docs? I'm honestly not sure.

Reply:
It might be good to specify which kind of virtualenv to use. I typically use Python 3's `venv` package, but it would be nice for contributors just to be able to follow specific guidelines.